Plugins/API/RainmeterAPI.h

```diff
@@ -179,6 +179,20 @@ LIBRARY_EXPORT void __cdecl RmLogF(void* rm, int level, LPCWSTR format, ...);
 /// </summary>
 LIBRARY_EXPORT BOOL __cdecl LSLog(int level, LPCWSTR unused, LPCWSTR message);
 
+/// <summary>
+/// Retrieves the option defined in a specific section of the skin file, allowing MeterStyle inheritance.
+/// </summary>
+/// <param name="rm">Pointer to the plugin measure</param>
+/// <param name="section">Section name to be read from</param>
+/// <param name="option">Option name to be read from the specified section</param>
+/// <param name="defValue">Default value if the option is not found or invalid</param>
+/// <param name="replaceMeasures">If true, replaces section variables in the returned string</param>
+/// <returns>Returns the option value as a string (LPCWSTR)</returns>
+#ifdef __cplusplus
+LIBRARY_EXPORT LPCWSTR __stdcall RmGetOption(void* rm, LPCWSTR section, LPCWSTR option, LPCWSTR defValue, BOOL replaceMeasures = TRUE);
+#else
+LIBRARY_EXPORT LPCWSTR __stdcall RmGetOption(void* rm, LPCWSTR section, LPCWSTR option, LPCWSTR defValue, BOOL replaceMeasures);

```

Comment:
Please remove this entirely and lets instead rename `RmGetOptionSafe` to be called `RmGetOption`. That way it will always be safe, there is no need to expose the unsafe version here.